core/src/main/scala/org/bitcoins/core/wallet/builder/TxBuilder.scala

```diff
@@ -570,7 +570,7 @@ object TxBuilder {
     */
   def calcSequenceForInputs(
       utxos: Seq[UTXOSpendingInfo],
-      isRBFEnabled: Boolean): Seq[TransactionInput] = {
+      defaultSequence: UInt32 = UInt32.zero): Seq[TransactionInput] = {
```

Comment:
why not make this `Policy.sequence` or something like that? I think users should have to explicitly enable things like RBF/CSV